### PR TITLE
Add precision to travel cost parameter in service area algorithms

### PR DESCRIPTION
**Pull request by DelazJ**
Should be less confusing for users (refs https://github.com/qgis/QGIS-Documentation/issues/3898)
Based on my experiments so it might need someone to check the code.

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

- [ ] Commit messages are descriptive and explain the rationale for changes
- [ ] Commits which fix bugs include `fixes #11111` in the commit message next to the description
- [ ] Commits which add new features are tagged with `[FEATURE]` in the commit message
- [ ] Commits which change the UI or existing user workflows are tagged with `[needs-docs]` in the commit message and contain sufficient information in the commit message to be documented
- [ ] I have read the [QGIS Coding Standards](https://docs.qgis.org/testing/en/docs/developers_guide/codingstandards.html) and this PR complies with them
- [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [ ] New unit tests have been added for core changes
- [ ] I have run [the `scripts/prepare-commit.sh` script](https://github.com/qgis/QGIS/blob/master/.github/CONTRIBUTING.md#contributing-to-qgis) before each commit

### CODE REVIEW

**Comment by DelazJ** [comment]
May I? Looking for confirmation since the released doc of this feature is probably wrong...

**Comment by nyalldawson** [comment]
I have two concerns here:
1. The description is looooong, and the dialog won't automatically wrap text. 
2. It's actually not "distance in map units", but rather "distance in the same units as the network layer" (not necessarily map units).

Could we put this descriptive text in the algorithm short help instead? (So that it's shown in the panel on the right)

**Comment by DelazJ** [comment]
> Could we put this descriptive text in the algorithm short help instead? (So that it's shown in the panel on the right)

Done! image

**Comment by nyalldawson** [comment]
Thanks, looks good to me!

File: python/plugins/processing/algs/help/qgis.yaml [hunk 1]
```diff
@@ -456,6 +456,16 @@ qgis:selectbyexpression: >
 
   For more information about expressions see the <a href ="{qgisdocs}/user_manual/working_with_vector/expression.html">user manual</a>
 
+qgis:serviceareafromlayer: >
+  This algorithm creates a new vector with all the edges or parts of edges of a network line layer that can be reached within a distance or a time, starting from features of a point layer.
+
+  The distance and the time (both refered to as "travel cost") must be specified respectively in the network layer units or in seconds.
```
Comment (nyalldawson) [inline python/plugins/processing/algs/help/qgis.yaml]:
```suggestion
  The distance and the time (both referred to as "travel cost") must be specified respectively in the network layer units or in seconds.
```

File: python/plugins/processing/algs/help/qgis.yaml [hunk 2]
```diff
@@ -456,6 +456,16 @@ qgis:selectbyexpression: >
...
+
+qgis:serviceareafrompoint: >
+  This algorithm creates a new vector with all the edges or parts of edges of a network line layer that can be reached within a distance or a time, starting from a point feature.
+
+  The distance and the time (both refered to as "travel cost") must be specified respectively in the network layer units or in seconds.
```
Comment (nyalldawson) [inline python/plugins/processing/algs/help/qgis.yaml]:
```suggestion
  The distance and the time (both referred to as "travel cost") must be specified respectively in the network layer units or in seconds.
```

**Comment by DelazJ** [comment]
@nyalldawson may I

**Comment by DelazJ** [comment]
Thanks

**Pull request merged**